valhalla/baldr/graphreader.h

```diff
@@ -26,7 +26,7 @@ namespace valhalla {
 namespace baldr {
 
 struct IncidentResult {
-  std::shared_ptr<incidents::IncidentsTile> tile;
+  std::shared_ptr<IncidentsTile> tile;
```

Comment:
renames and the move the helper function to the source file. maybe we can delete this helper function as well since its never used directly cc @PureW @danpat